doc/voice-api.md

```diff
@@ -0,0 +1,51 @@
+# Almond Voice
+
+Currently, Almond Voice provides two REST endpoints that provides TTS and STT functionality for Almond-based services, both hosted at [voice.almond.stanford.edu](https://voice.almond.stanford.edu). Support for websocket-based streaming will be added in the future.
+
+## Speech-to-text
+
+### Request
+
+```
+POST /rest/stt
+Host: voice.almond.stanford.edu
+Content-Type: multipart/form-data
+```
+
+Where the body of the request contains a `.wav` file. The file does not need to have a specific sample rate, as the server automatically resamples submitted audio.
```

Comment:
Doesn't expect a form upload with a specific name, I think. Will ask for correct mime-type for wav. Don't think we will handle direct wav uploads without multipart wrapper. 

Will add bit-depth (16) and signedness (LE) requirements. We're not handling those variations as my understanding is that they are very common.

Reply:
S16 PCM is by far the most common non-pro audio format out there, so it's totally fine to only support that, we just need to spell it out because there is code that produces 8 bit sound.

Reply:
Ok, added in latest commit